src/components/PostListPage/PostListHeader.js

```diff
@@ -0,0 +1,16 @@
+import { CLASSNAME } from "../../utils/constants.js";
+
+export default function PostListHeader({ $target }) {
+  const $header = document.createElement("div");
+  $header.className = CLASSNAME.POST_LIST_HEADER_DIV;
+
+  $target.appendChild($header);
+
+  this.render = () => {
+    $header.innerHTML = `
+	  <h3 class=${CLASSNAME.POST_LIST_HEADER}>김정호의 노션</h3>
```

Comment:
이런 종류의 코드를 작성할 때는 마크업 자체가 깨질 수 있기 때문에

```suggestion
	  <h3 class="${CLASSNAME.POST_LIST_HEADER}">김정호의 노션</h3>
```

double quote로 감싸주기를 권장합니다.